library/core/src/macros/mod.rs

```diff
@@ -1452,6 +1452,18 @@ pub(crate) mod builtin {
         /* compiler built-in */
     }
 
+    /// Expands all `#[cfg]` and `#[cfg_attr]` attributes in the code fragment it's applied to.
```

Comment:
We should add an example here prior to stabilization, but it doesn't need to block this PR.